### DISCUSSION TITLE
Add syntax highlighting for proto and proto3 files

### DISCUSSION THREAD

**Discussion by IvanPizhenko** [community/community#102665] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Just noticed that proto and proto3 Protobuf files are not syntx-highlighted. Please add this. It will make easier to read them from web UI.

**Comment by spenserblack** [comment]
Language detection and serving the appropriate syntax highlighter is done by [linguist](https://github.com/github-linguist/linguist). If your language isn't supported, but it is reasonably popular, you can make a PR to add support. If your language *is* supported, but you're using an unsupported filename or extension, you can make a PR to register that filename or extension for your language. However, there are popularity requirements -- you can't add a filename or extension that very few people use. If that happens, you can add an override *to your own repository* via `.gitattributes` like so:
```gitattributes
# .gitattributes
path/to/file*.ext linguist-language=Protocol-Buffer
```

I don't know anything about Protobuf files, so the above example may be wrong. But I *think* linguist (and therefore GitHub) already supports your files due to [these lines](https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/lib/linguist/languages.yml#L5428-L5452). You'll want to view the [overrides documentation](https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/docs/overrides.md) for more info.
